ql/src/java/org/apache/hadoop/hive/ql/parse/StorageFormat.java

```diff
@@ -48,9 +51,14 @@ public class StorageFormat {
 
   public enum StorageHandlerTypes {
     DEFAULT(),
-    ICEBERG("\'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler\'",
```

Comment:
wow, what was this "\" used for and why?

Reply:
I'm not sure about the original intention. IntelliJ asks me to remove the backslash and it seems to be safe to remove.